docs/SX126x_SX127x_INCOMPATIBILITY.md

```diff
@@ -0,0 +1,13 @@
+# mLRS Documentation: SX126x and SX127x Incompatibility #
+
+([back to main page](../README.md))
+
+mLRS hardware using the sx126x and sx127x LoRa chipsets are not compatible with each other, and cannot be operated together (they don't bind).
+
+This is for two reasons:
+- the SF6 spreading factor, which is used by mLRS for the 19 Hz mode, is incompatible for the sx126x and sx127x chipsets. 
+- the sx127x does not support the SF5 spreading factor which is used for the mLRS 31 Hz mode.
+
```

Comment:
Perhaps:

This is for two reasons:

1. The 31 Hz mode uses spreading factor 5 (SF5), which is only available on the SX126x chipset.
2. The 19 Hz mode uses spreading factor 6 (SF6), which was modified on the SX126x chipset making it incompatible with the SX127x chipset.

Reply:
done